ale/drivers/base.py

```diff
@@ -655,7 +659,7 @@ def _semimajor(self):
           Semimajor axis of the target body
         """
         rad = spice.bodvrd(self.target_name, 'RADII', 3)
-        return rad[1][1]
+        return rad[1][0]
```

Comment:
Are these new changes?

Reply:
yeah, I seem to only get the correct semimajor/semininor this way. The docs for cspice are not clear about what the 3 element array holds, but when using CTX images 0th and 2nd element had the right radii.

Reply:
:+1: